### DISCUSSION TITLE
96/24?

### DISCUSSION THREAD

**Discussion by 59volvo** [SamDel/ChromeCast-Desktop-Audio-Streamer#99] [Ideas]
Is there a 96k/24bit option for a higher fidelity stream - since the chromecast is capable of 96/24?

**Comment by SamDel** [comment]
~~In [3.3](https://github.com/SamDel/ChromeCast-Desktop-Audio-Streamer/releases/tag/v3.3) the stream is always converted to 44,1kHz/16bit.~~

~~In [version 3.3.3](https://github.com/SamDel/ChromeCast-Desktop-Audio-Streamer/files/6068550/Setup.3.3.3.zip) (development version) when wav is selected it streams the audio in the samplerate of your soundcard. It's always converted to 16bit in 3.3.3.~~

~~In [Setup 3.3.4.zip](https://github.com/SamDel/ChromeCast-Desktop-Audio-Streamer/files/6073895/Setup.3.3.4.zip) the samplerate of your soundcard is used. It's converted to (max) 24bit in 3.3.4.~~

~~Can you try if 3.3.4 works for you? Is your soundcard capable of 96kHz?~~

See comment below.

**Comment by SamDel** [comment]
It's easier to test with [Setup 3.3.5.zip](https://github.com/SamDel/ChromeCast-Desktop-Audio-Streamer/files/6074405/Setup.3.3.5.zip). I added the different Wav formats in the stream format dropdown:

- Wav (44100/16bit): The captured stream is resampled to 44,1kHz and converted to 16bit. (same as is version 3.3 and earlier versions)
- Wav (Soundcard's sample rate/16bit): The captured stream is NOT resampled, only converted to 16bit.
- Wav (Soundcard's sample rate/24bit): The captured stream is converted to 24bit.
- Wav (Soundcard's sample rate/32bit): The captured stream is converted to 32bit. (gives cracklings for me!)

The sample rate that's used is logged like "Stream format set to Pcm 48000 24 bit" on the log tab.

**Comment by b0ogdan** [comment]
Thank you very much for making this app and keeping it updated.
Is there a way to add the option to select a particular app to stream from? Like for example if i want to cast spotify on my smart speakers but at the same time i want o listen to a youtube video on only on my desktop`s speakers.

**Comment by SamDel** [comment]
Thanks! Maybe it's possible with multiple sound output devices (add a virtual audio device?), see method 2 [here](https://www.minitool.com/news/play-sound-multiple-output-devices-windows-10.html), and then select the right recording device on the options tab of this app. I didn't try. Let me know!
